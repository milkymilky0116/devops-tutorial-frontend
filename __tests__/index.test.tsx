/**
 * @jest-environment jsdom
 */
import { render, screen } from "@testing-library/react";
import Home from "@/pages/index";

describe("Home", () => {
  it("render a home", () => {
    const { container } = render(<Home />);

    const home = screen.getByText("Test");

    expect(home).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
