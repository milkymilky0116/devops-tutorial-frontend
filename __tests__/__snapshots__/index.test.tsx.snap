// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home render a home 1`] = `
<div>
  <div>
    Hello Everyone!
  </div>
</div>
`;
